content/v2/csidriver/features/powerscale.md

```diff
@@ -1,570 +1,659 @@
+4. If Quotas are enabled in the driver, it is required to add the Quota ID to the description of the NFS export in this format: 
+`CSI_QUOTA_ID:sC-kAAEAAAAAAAAAAAAAQEpVAAAAAAAA`
+5. Quota ID can be identified by querying the PowerScale system.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: isilonstaticpv
+  namespace: default
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: isilon
+  csi:
+    driver: csi-isilon.dellemc.com
+    volumeAttributes:
+        Path: "/ifs/data/csi/isilonvol"
+        Name: "isilonvol"
+        AzServiceIP: 'XX.XX.XX.XX'
+    volumeHandle: isilonvol=_=_=652=_=_=System=_=_=pscale-cluster
+  claimRef:
+    name: isilonstaticpvc
+    namespace: default
+```
+
+3. Create PersistentVolumeClaim to use this PersistentVolume.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: isilonstaticpvc
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+        requests:
+          storage: 5Gi
+  volumeName: isilonstaticpv
+  storageClassName: isilon           
+```
+
+4. Then use this PVC as a volume in a pod.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+    name: static-prov-pod
+spec:
+    containers:
+      - name: test
+        image: docker.io/centos:latest
+        command: [ "/bin/sleep", "3600" ]
+        volumeMounts:
+          - mountPath: "/data0"
+            name: pvol
+    volumes:
+      - name: pvol
+        persistentVolumeClaim:
+            claimName: isilonstaticpvc
+```
+
+5. After the pod becomes `Ready` and `Running`, you can start to use this pod and volume.
+
+## PVC Creation Feature
+
+Following yaml content can be used to create a PVC without referring any PV.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: testvolume
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+        requests:
+          storage: 5Gi
+  storageClassName: isilon           
+```
+
+## Volume Snapshot Feature
+
+The CSI PowerScale driver version 2.0 and later supports managing v1 snapshots. 
+
+In order to use Volume Snapshots, ensure the following components have been deployed to your cluster:
+
+- Kubernetes Volume Snapshot CRDs
+- Volume Snapshot Controller
+
+> For general use, update the snapshot controller YAMLs with an appropriate namespace before installing. For
+> example, on a Vanilla Kubernetes cluster, update the namespace from default to kube-system before issuing the
+> kubectl create command.
+
+### Volume Snapshot Class
+
+During the installation of CSI PowerScale driver version 2.0 and higher, no default Volume Snapshot Class will get created.
+
+Following are the manifests for the Volume Snapshot Class:

```

Comment:
Insert "The" before "Following". Remove initial uppercase on "Following"

Reply:
Addressed the comment